internal/generator/discoverTemplate.go

```diff
@@ -57,7 +57,7 @@ func discover_{{ .MethodName }}(service *runner.Service) error {
 		},
 		Lookups: []runner.LookupDiscovery{
 {{- range .Lookups }}
-			runner.LookupDiscovery{
```

Comment:
you need the types here

Reply:
why? is it a golang version thing? the linter was complaining about the redundant type specification in my use case

Reply:
TIL